dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2093,11 +2097,63 @@ def test_split_row_groups_pyarrow(tmpdir):
     )
 
     ddf3 = dd.read_parquet(
-        tmp, engine="pyarrow", gather_statistics=True, split_row_groups=True
+        tmp,
+        engine="pyarrow",
+        gather_statistics=True,
+        split_row_groups=True,
+        chunksize=1,
     )
     assert ddf3.npartitions == 12
 
     ddf3 = dd.read_parquet(
         tmp, engine="pyarrow", gather_statistics=True, split_row_groups=False
     )
     assert ddf3.npartitions == 4
+
+
+@pytest.mark.parametrize("metadata", [True, False])
+@pytest.mark.parametrize("chunksize", [None, 1024, 4096, "1MiB"])
+def test_chunksize(tmpdir, chunksize, engine, metadata):
+    check_pyarrow()
```

Comment:
Do we need the pyarrow check here?  Or can we rely on the engine fixture to iterate over both.

Reply:
nope - my mistake

Reply:
My mistake agian.  It seems that pyarrow is required for the read-phase of the test, because we want to write out a single partition with multiple row groups (not currently possible with the fastparquet backend)